### ISSUE TITLE
Error 404 after upgrade to 3.3

### ISSUE THREAD

**Issue by cerelli** [Laravel-Backpack/Settings#59]
# Bug report
Page not found. 404

### What I did:
Open menu setting: http://localhost:8000/admin/setting

### Backpack, Laravel, PHP, DB version:
3.3, 5.5, 7.0, MySQL [10.1.28-MariaDB]

I notice that my vendo/backpack/settings/composer.json is:
`...
    "require": {
        "illuminate/support": "~5.1",
        "php" : ">=5.3.0",
        "backpack/crud": "^3.0.6"
    },
...
`

**Comment by davemin** [comment]
same issue

**Comment by tabacitu** [comment]
Hi guys,

The PR that fixes this has been merged on Nov 22 - just tested it again and it fixes the problem. 

However, as of today Backpack\Demo was NOT using that new version of Backpack\Settings. If you were having this problem with the demo, that was the problem, I've now pushed an update to the demo for that.

In case you're having this problem in your own projects, a ```composer update``` will fix it for you guys.

Cheers!
